esp/scm/ws_workunits.ecm

```diff
@@ -1434,7 +1434,8 @@ ESPresponse [exceptions_inline] WUQueryFilesResponse
 
 ESPrequest WUQueryDetailsRequest
 {
-    string QueryId;
+    [depr_ver("1.61")] string QueryId;
+    [min_ver("1.61")] string QueryIdOrAlias;
```

Comment:
The version may be needed if a client wants to know which request parameter can be used for a specific version of the client.